docs/data/joy/components/select/select.md

```diff
@@ -136,11 +136,12 @@ The `FormControl` automatically generates a unique id that links the select with
 Alternatively, you can do it manually by targeting the button slot:
 
 ```jsx
-<label htmlFor="unique-id">Label</label>
+<label htmlFor="select-button" id="select-label">Label</label>
 <Select
   componentsProps={{
     button: {
-      id: 'unique-id',
+      id: 'select-button',
+      'aria-labelledby': 'select-label select-button',
     }
   }}
 >
```

Comment:
I have updated the instructions to be correct.

Reply:
Why does it need both? I tested with VoiceOver and it announced the label with `htmlFor` alone.

Reply:
@siriwatknp This demo is not working on my end. See https://github.com/mui/material-ui/pull/17892. I have the following with VoiceOver:

<img width="1047" alt="Screenshot 2022-09-19 at 01 28 27" src="https://user-images.githubusercontent.com/3165635/190932719-bf4c6fc0-71ef-4faf-a0f0-ad81d36041fd.png">

https://mui.com/joy-ui/react-select/#accessibility

and 

<img width="1152" alt="Screenshot 2022-09-19 at 01 32 08" src="https://user-images.githubusercontent.com/3165635/190932828-6a15e92e-4853-456a-a7d5-b2f81a923c48.png">

https://deploy-preview-34116--material-ui.netlify.app/joy-ui/react-select/#accessibility

https://www.w3.org/WAI/ARIA/apg/example-index/combobox/combobox-select-only.html uses aria-labelledby too.

Reply:
@siriwatknp Do you confirm the same behavior or are you observing something different?

Reply:
You are right. I tested it again on https://mui.com/joy-ui/react-select/#accessibility which does not announce the "Favorite pet Dog".

Reply:
Awesome, happy that we are observing the same thing :)